src/Sentry.Maui.CommunityToolkit.Mvvm/CtMvvmMauiElementEventBinder.cs

```diff
@@ -0,0 +1,149 @@
+using System.Windows.Input;
+using CommunityToolkit.Mvvm.Input;
+
+namespace Sentry.Maui.CommunityToolkitMvvm;
+
+/// <summary>
+/// Scans all elements for known commands that are implement
+/// </summary>
+public class CtMvvmMauiElementEventBinder(IHub hub) : IMauiElementEventBinder
```

Comment:
question(naming): is `CtMvvm` a well-known abbreviation for `CommunityToolkit.Mvvm`?

To me this type name doesn't read like idiomatic .NET.
Unless `CtMvvm` is a super common abbreviation used by the community ... but I personally have never encountered it so far.

On the other hand, a full `CommunityToolkitMvvmMauiElementEventBinder` is a bit unwieldy.

Could we get away with making this type `internal` for now?
Since I believe it's meant to be added to DI via the `MauiAppBuilder`-Extension method.
Or does user code need to refer to it?

Also:
From what I see, the other `IMauiElementEventBinder`-derived types we have are prefixed with `Maui`.
Does it make sense to go with `MauiCommunityToolkitMvvmEventsBinder` instead?
And/or make the type `internal`?

What do you think?

Reply:
Renamed - I don't want to debate naming.  Let's just get it done.